main.go

```diff
@@ -43,6 +43,7 @@ func main() {
 	flagSet.Bool("skip-provider-button", false, "will skip sign-in-page to directly reach the next step: oauth/start")
 	flagSet.Bool("skip-auth-preflight", false, "will skip authentication for OPTIONS requests")
 	flagSet.Bool("ssl-insecure-skip-verify", false, "skip validation of certificates presented when using HTTPS")
+	flagSet.Duration("flush-interval", time.Duration(1)*time.Second, "period between response flushing when streaming responses")
```

Comment:
I think this should default to `0` rather than `1` since this technically changes the behaviour. I appreciate it improves the behaviour but someone may be relying on the old behaviour. (You also won't need any of that casting stuff there since `Durations` are int64 anyway)

```suggestion
	flagSet.Duration("flush-interval", 0, "period between response flushing when streaming responses")
```

Reply:
I understand keeping things backwardly compatible, but TBH the existing behaviour sucks and I would consider a bug. When proxying logs, it doesn't even flush on newlines - you just get the response stopping halfway through a word.

Reply:
I'd strongly urge the default to be 1s (from experience of using it) but if you still want it to be 0, let me know here and I'll change it.

Reply:
I think you are right, the existing behaviour sucks, maybe I'm being too cautious but I think for now I'd like to make the default 0, with the view to changing it to 1 after the change has had some further testing and we have given warning that it will be changing (we should put a note in the release note that the plan is to flip the default in the next release). WDYT @agentgonzo @syscll ?

Reply:
In honesty, I think you're being too cautious. :-)
The existing behaviour is bad - without putting a too finer point on it, if people are _relying_ on this (bad) behaviour, then something is very broken in what they are trying to do achieve with their deployments. I see no reasonable situation where you would _want_ the existing behaviour